pkg/watcher/resource-event-handler/pod.go

```diff
@@ -34,9 +34,9 @@ func (p *podEventHandler) GetResourceObject() runtime.Object {
 }
 
 func (p *podEventHandler) OnAdd(obj interface{}) {
-	glog.V(4).Infof("OnAdd(): pod %v", obj)
+	log.Debug().Msgf("OnAdd(): pod %v", obj)
 	pod := obj.(*kapi.Pod)
-	glog.Infof("OnAdd(): namespace %s name %s", pod.Namespace, pod.Name)
+	log.Info().Msgf("OnAdd(): namespace %s name %s", pod.Namespace, pod.Name)
 
 	if !utils.PodWantsNetwork(pod) {
 		return
```

Comment:
add perhaps a debug log saying this pod does not require network

Reply:
same maybe for some of the returns below, in case we need to understand why a pod didnt end up in the daemon queue